tests/scripts/test_capture_native_profile_bounds.py

```
@@ -0,0 +1,888 @@
+
+
+# ---------------------------------------------------------------------------
+# http_json: redirect rejection
+# ---------------------------------------------------------------------------
+
+@pytest.mark.parametrize("mod", _MODULES, ids=_MODULE_IDS)
+def test_http_json_redirect_not_followed(fake_server, mod):
+    # The server sends a 302 redirect.  Our http_json should not follow it.
+    srv = fake_server(mode="redirect", redirect_target="http://evil.example.com/v1/models")
+    headers = {"Content-Type": "application/json"}
+    status, body = mod.http_json(f"{srv.base_url}/start_profile", headers=headers, payload={}, max_response_bytes=4096)
```

Ruff (0.16.1)

[warning] Unpacked variable `status` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)